modal/cli/launch.py

```diff
@@ -61,6 +63,16 @@ def jupyter(
     volume: Optional[str] = None,  # Attach a persisted `modal.Volume` by name (creating if missing).
     detach: bool = False,  # Run the app in "detached" mode to persist after local client disconnects
 ):
+    console = make_console()
+    console.print(
+        rich.panel.Panel(
+            (
+                "[underline]Try Modal Notebooks! [link=https://modal.com/notebooks]modal.com/notebooks[/link][/underline]\n"
```

Comment:
Nit:

```suggestion
                "[link=https://modal.com/notebooks]Try Modal Notebooks! "
                "modal.com/notebooks[/link]\n"
```

![Screenshot 2025-08-01 at 11 43 02 AM](https://github.com/user-attachments/assets/d6a047fc-e554-4766-9640-603a190e279f)